tutorials/tour/singleton-objects.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: tutorial
+title: Singleton Objects
+
+disqus: true
+
+tutorial: scala-tour
+num: 28
+---
+
+Methods and values that aren't associated with individual instances of a [class](classes.html) belong in *singleton objects*, denoted by using the keyword `object` instead of `class`.
+
+    package test
+
+    object Blah {
+      def sum(l: List[Int]): Int = l.sum
+    }
+
+This `sum` method is available globally, and can be referred to, or imported, as `test.Blah.sum`.
+
+Singleton objects are sort of a shorthand for defining a single-use class, which can't directly be instantiated, and a `val` member at the point of definition of the `object`, with the same name. Indeed, like `val`s, singleton objects can be defined as members of a [trait](traits.html) or class, though this is atypical.
+
+A singleton object can extend classes and traits. In fact, a [case class](case-classes.html) with no [type parameters](generic-classes.html) will by default create a singleton object of the same name, with a [`Function*`](http://www.scala-lang.org/api/current/scala/Function1.html) trait implemented.
+
+## Companions ##
+
+Most singleton objects do not stand alone, but instead are associated with a class of the same name. The “singleton object of the same name” of a case class, mentioned above, is an example of this. When this happens, the singleton object is called the *companion object* of the class, and the class is called the *companion class* of the object. This is so common that it's safe to refer to singleton objects as “companion objects” informally, without much risk of confusion.
```

Comment:
I don't agree with this last sentence; it seems incorrect and misleading to me to call something a "companion object" unless it actually is one. Let's not encourage this.